.github/workflows/step_test.yaml

```diff
@@ -73,12 +73,11 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
           allow-prereleases: ${{ matrix.pre || false }}
-        # TODO: Replace with editable install once configure presets are available
-      - name: Install python bindings
-        run: >
-          python3 -m pip install
-          ${{ matrix.pre && '--pre' }}
-          .[test]
+      - name: Install dependencies
+        run: |
+          python3 -m pip install pip-tools
+          pip-compile --extra=test --strip-extras pyproject.toml ${{ matrix.pre && '--pre' }}
```

Comment:
Why does this PR generate requirements.txt here? To lock package versions?

Reply:
`pip` does not have a mode to install only dependencies, and that is the closest approach I've found to do that. What I wanted to be covered there is the case where the Spglib is built without `scikit-build-core` being involved. Weird though that it didn't error out in the previous iterations where I didn't include `configure_file(_version.py.in spglib/_version.py)`

Reply:
Although I feel tricky, it makes sense. Thanks!

Reply:
I fully agree. Eventually I hope the [`skbuild-cli`](https://github.com/scikit-build/scikit-build-cli) will be implemented, at which point we wouldn't need to care about such difference.